core/frontend/meta/context_object.js

```diff
@@ -26,6 +26,13 @@ function getContextObject(data, context) {
         chosenContext = data.tag;
     } else if (_.includes(context, 'author') && data.author) {
         chosenContext = data.author;
+    } else if (_.includes(context, 'page') && data.post) {
```

Comment:
I'm not sure that this if statement is necessary - did you test if it gets executed?

Logically it is the exact same as:

`if ((_.includes(context, 'page') || _.includes(context, 'amp')) && data.post) {`

So I can't see that this is needed

Reply:
@ErisDS Yeah you are right, my bad. I had overlooked the first statement and just saw the
 `_.includes(context, 'page') && data.page` and thought that I had missed it. Fixed it and updated the PR.